tox.ini

```diff
@@ -352,6 +353,7 @@ commands =
 # run subsets of the tests for particular library versions
     ddtracerun: pytest {posargs} tests/commands/test_runner.py
     test_utils: pytest {posargs} tests/contrib/test_utils.py
+    test_logging: pytest {posargs} tests/contrib/logging/
```

Comment:
does this get run via the `tracer` tests?

Reply:
Moved it to `tests/contrib` so it is ignored by the `tracer` environment.